### PR TITLE
Cleans up page element alignments and makes the page more mobile friendly

### PR DESCRIPTION
**Pull request by jasoryeh**
Just a bit of changes to make the page more aligned so the images aren't flowing off the page on mobile, and so there isn't a random hovering bar when you first load the page.

- Clean up *slightly* misaligned elements
- Adds the nav-bar to config editor
- Makes page more mobile friendly with more media queries
- Makes navbar completely transparent when not scrolled, and adds Geyser logo when scrolled

### CODE REVIEW

File: config_editor.html [hunk 1]
```diff
@@ -7,9 +7,17 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css" integrity="sha256-aPeK/N8IHpHsvPBCf49iVKMdusfobKo2oxF8lRruWJg=" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma-switch@2.0.0/dist/css/bulma-switch.min.css" integrity="sha256-jCV/cXwP13w0GNHLgFx6SFgTNAvJPvS5MIhuE30Ng08=" crossorigin="anonymous">
     <link rel='stylesheet' type='text/css' media='screen' href='css/config-editor.css'>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+        nav * {
+            color: black !important;
+        }
```
Comment (rtm516) [inline config_editor.html]: Can this be pulled into the `css/config-editor.css` file?

File: css/style.css [hunk 2]
```diff
@@ -51,19 +79,31 @@ h1.text-center {
...
-.section2 .col {
-	padding: 80px 30px 30px 10px;
-}
+.section2 .col {}
 
-.section2 img {
-	height: 200px;
-	width: 400px;
-}
+.section2 img {}
```
Comment (rtm516) [inline css/style.css]: These can be removed

File: css/style.css [hunk 3]
```diff
@@ -92,13 +131,9 @@ h1.text-center {
...
 .section4 .col {
 	color: white;
-	padding: 80px 30px 30px 10px;
 }
 
-.section4 img {
-	max-width: 600px;
-	width: 97vw;
-}
+.section4 img {}
```
Comment (rtm516) [inline css/style.css]: As above

File: index.html [hunk 4]
```diff
@@ -195,18 +187,26 @@ <h6 class="mb-0">&copy 2020 GeyserMC</h6>
...
+                  $('.nav-logoimg').css("opacity", '1.0');
                 } else {
-                  $(".navbar").css("background-color", "rgba(255, 255, 255, 0.1)");
+                  $(".navbar").css("background-color", "rgba(255, 255, 255, 0.0)");
+                  $(".navbar").css("box-shadow", "0 .5rem 1rem rgba(0, 0, 0, 0.0)");
                   $(".navbar").css("margin", "10px");
-                  $(".navbar").css("padding", "10px 10px 10px 10px");
+                  $(".navbar").css("padding", "5px 10px 5px 10px");
+                  $(".navbar").css("border-radius", "25px");
+                  $('.nav-logoimg').css("opacity", '0.0');
```
Comment (rtm516) [inline index.html]: Can `.navbar` and `.nav-logoimg` be pulled into variables to prevent searching for the element every time?

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Comment by rtm516** [comment]
There are a few issues caused by this PR
 - The homepage on a 1920x1080 device isn't full width so leaves a white gap on the right-hand side
 - The buttons on the config editor page
  - Don't line up
  - Text between is invisible on the background as its white on white
 - The navigation on the config editor page
   - Has no hover effects on the buttons
   - Isn't full width of the page
 - The sections in the config editor now have rounded corners so would look better with gaps in-between.
 - The navbar on the homepage
   - Drifts up slightly on load
   - Moves when you start scrolling

**Comment by jasoryeh** [comment]
> There are a few issues caused by this PR
> 
> * The homepage on a 1920x1080 device isn't full width so leaves a white gap on the right-hand side ✔
> * The buttons on the config editor page don't line up ✔
> * Text between is invisible on the background as its white on white ✔
> * The navigation on the config editor page
>   * Has no hover effects on the buttons ✔
>   * Isn't full width of the page ❓
> * The sections in the config editor now have rounded corners so would look better with gaps in-between. ✔
> * The navbar on the homepage 
>   * Drifts up slightly on load ✔
>   * Moves when you start scrolling 💯This is kind of purposeful, to make the navbar take a not so giant chunk of screen real estate. I edited the animations to make the move and shrink smoother.

Fixed most of those issues, and fixed some missing things while I was at it.

File: config_editor.html [hunk 5]
```diff
@@ -24,9 +27,52 @@
...
+                        <i class="fas fa-sliders-h"></i>&nbsp;<span>Config Editor</span>
+                    </a>
+                </li>
+                <!--
+                <li class="nav-item">
+                    <a class="nav-link" href="https://patreon.com/GeyserMC"><i class="fab fa-patreon"></i> Donate</a>
+                </li>
+                -->
+            </ul>
+        </div>
```
Comment (rtm516) [inline config_editor.html:71]: Could this be wrapped in a `<div class="container">` to pull the logo and buttons in so it doesn't look as strange with the rest of the page?

File: config_editor.html [hunk 6]
```diff
@@ -24,9 +27,52 @@
     <meta name="theme-color" content="#ffffff">
 </head>
 <body>
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
```
Comment (rtm516) [inline config_editor.html:30]: This doesn't have a mobile navigation bar making it impossible to go anywhere else but 'back' on mobile

**Review by rtm516 — CHANGES_REQUESTED** [review]
Nice work so far, but along with these few minor last changes could you:
 - Add a background to the mobile nav when on the first slide on the homepage
 - Look into the horizontal scrollbar sometimes visible on mobile device sizes, looks like it may be caused by the wider images not being scaled.

File: config_editor.html (hunk 6 above)
Comment (jasoryeh) [reply config_editor.html:30]: Resolved with grey background for mobile users.

File: config_editor.html (hunk 5 above)
Comment (jasoryeh) [reply config_editor.html:71]: Unfortunately doing so will throw off things inside the nav, and things start to move to the next line

Comment (jasoryeh) [reply config_editor.html:71]: Gotta play around with it, but the other css we had was sort of wonky already so a lot of the changes I proposed were trying to fix up the weird styling.

Comment (jasoryeh) [reply config_editor.html:71]: Resolved with invert when mobile

**Review by rtm516 — APPROVED** [review]

**Pull request merged**